app/javascript/utilities/search/index.js

```diff
@@ -70,7 +70,7 @@ export function getInitialSearchTerm(querystring) {
     matches !== null && matches.length === 2
       ? decodeURIComponent(matches[1].replace(/\+/g, '%20'))
       : '';
-  const query = filterXSS(rawSearchTerm);
+  const query = filterXSS(rawSearchTerm) || '';
```

Comment:
This fixes an issue where it became a string `"undefined"` instead of just an empty string.